reverse_proxy/defaults/main.yml

```diff
@@ -5,3 +5,7 @@ zone: tre.example.org
 # cran mirror
 cran_mirror: https://cran.ma.imperial.ac.uk/
 cran_regex: ^/(bin/windows|src)/contrib/(4\.1|4\.3|3\.6)*/*[a-zA-Z0-9_\-\.]*(\.tar\.gz|\.zip|PACKAGES)$
+
+# conda-forge
+conda_repo_regex: ^/conda-forge/(linux-64|linux-aarch64|win-64|noarch)/(current_)*repodata.json$
```

Comment:
`(current_)*`
Should this be `?` instead of `*`?

Reply:
Good point!